Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -340,7 +341,11 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor {
    * This will cause more tiles to be loaded, but helps to avoid holes and
    * provides a more consistent mesh, which may be helpful for physics.
```

Comment:
```suggestion
   * provides a more consistent mesh, which may be helpful for physics.
   * 
   * Note that this will always be disabled if UseLodTransitions is set to true.
```